### DISCUSSION TITLE
Handling Multiple Marketplaces?

### DISCUSSION THREAD

**Discussion by TechyChap** [abuzuhri/Amazon-SP-API-CSharp#445] [General]
Wondering what is the recommendation for handling multiple marketplaces. We are predominantly looking at Listings but also moving into Orders.

Should we have one connection and pass the marketplace ID and URL?
Or should see have one connection per marketplace with predefined marketplace/URL?

**Comment by abuzuhri** [comment]
Its depending in marketplaces in same region or not , also on endpoint .

For order endpoint its take list of marketplace (must be in same region)  (_**MarketplaceIds**_) , and its should be ok no problem with it
